### DISCUSSION TITLE
Obtainium 1.0.4 false update notifications

### DISCUSSION THREAD

**Discussion by bbrockie** [ImranR98/Obtainium#1454] [General]
I keep getting notifications about updates that Revanced has a update to, but it is for a older version. I have already updated Revanced to latest versions. Can confirm this on 2 of my devices.

**Comment by bbrockie** [comment]
Obtainium keeps notifying that there is a update to Revanced v1.19.0.
Revanced has been updated to v1.19.1.
How can I manually skip this version in Obtainium?

**Reply by DwainZwerg** [reply]
You could make the app a track-only source. However, this seems to be a bug. How about opening a new issue?

**Comment by bbrockie** [comment]
![Screenshot_20240306-233848.jpg](https://github.com/ImranR98/Obtainium/assets/148401570/cf1b3278-d427-450c-8fed-4d8179938f55)

**Comment by ImranR98** [comment]
This is happening because Obtanium sorts releases by the release date and picks the newest one (it doesn't just use the existing order of releases provided by the GitHub API). So you could either turn on the "retain release order..." toggle or the "use latest asset..." toggle (since the actual APK uploads are also in the right order).

You should also have been able to use the "verify latest" toggle, but it currently is broken due to issue #1449.
